airbyte-webapp/src/components/LabeledControl/ControlLabels.tsx

```diff
@@ -29,9 +30,19 @@ const ControlLabels: React.FC<React.PropsWithChildren<ControlLabelsProps>> = (pr
       nextLine={props.nextLine}
     >
       {props.label}
+      {props.infoMessage && (
+        <InfoTooltip className={styles.tooltip} placement="top-start">
+          {props.infoMessage}
+        </InfoTooltip>
+      )}
+      {props.optional && (
+        <Text size="sm" className={styles.optionalText}>
+          <FormattedMessage id="form.optional" />
+        </Text>
+      )}
```

Comment:
To me, the `<Label>` component felt like a more generic component that I wanted to keep pretty simple - basically it has the "content" (which are just the children you pass in), a "message", which is what gets displayed after the label content, and a couple other customization props. And this Label is used in this simple way in a lot of places.

The `ControlLabels` component to me felt like something that adds on top of the simple label: it offers the ability to add an info tooltip and an `Optional` string into the label "content". These two things felt like a more specific requirement for Form field labels specifically, which is where the `ControlLabel` component tends to get used, so I added those here. The `Optional` string specifically felt like it didn't make a ton of sense to add to `Label` since I think there are places that Label is used where optional strings don't really make sense, like Settings page.

However, I don't feel too strongly about this so I'm happy to consolidate all of this into the `Label` component if you think we should!